data_steward/analytics/tools/concept_success_rates_visualizations.py

```diff
@@ -190,12 +190,12 @@
                   'cpmc_ucd', 'cpmc_uci', 'cpmc_ucsd', 'cpmc_ucsf', 'cpmc_usc', 'ecchc',
                   'hrhc', 'ipmc_northshore', 'ipmc_nu', 'ipmc_rush', 'ipmc_uchicago',
                   'ipmc_uic', 'jhchc', 'nec_bmc', 'nec_phs', 'nyc_cornell', 'nyc_cu',
-                  'nyc_hh', 'pitt', 'pitt_temple', 'saou_lsu', 'saou_uab',
-                  'saou_uab_hunt', 'saou_uab_selma',
+                  'nyc_hh', 'pitt', 'pitt_temple', 'saou_lsu', 'saou_tul', 'saou_uab',
+                  'saou_uab_hunt', 'saou_uab_selma', 'saou_umc',
                   'saou_ummc', 'seec_emory', 'seec_miami', 'seec_morehouse',
                   'seec_ufl', 'syhc', 'tach_hfhs', 'trans_am_baylor',
```

Comment:
Do you want to try using the `bq_utils.get_hpo_info()` for this? Since we are now using `utils.bq` going forward, I think it would help to migrate the function 'get_hpo_info' as well.

Reply:
I created a ticket (EDQ-421) to address this issue. I will be sure to address it after I finish generating the notebooks for the NIH metrics.

Reply:
@nishanthpp93 Do you have any examples of this function call being used previously? I got the following error and do not want to potentially set the incorrect project.

Here is the exact error I received from running 'bq_units.get_hpo_info()':

> ---------------------------------------------------------------------------
> RuntimeError                              Traceback (most recent call last)
> <ipython-input-15-974a942996a7> in <module>()
> ----> 1 bq_utils.get_hpo_info()
> 
> /Users/Noah/Coding/aou_curation/curation/data_steward/bq_utils.py in get_hpo_info()
>     926 def get_hpo_info():
>     927     hpo_list = []
> --> 928     project_id = app_identity.get_application_id()
>     929     hpo_table_query = bq_consts.GET_HPO_CONTENTS_QUERY.format(
>     930         project_id=project_id,
> 
> /Users/Noah/Coding/aou_curation/curation/data_steward/app_identity.py in get_application_id()
>      17     else:
>      18         raise RuntimeError(
> ---> 19             '{} is not set.  Set and retry.'.format(GOOGLE_CLOUD_PROJECT))

RuntimeError: GOOGLE_CLOUD_PROJECT is not set.  Set and retry.

Reply:
You might have to set the environment variable GOOGLE_CLOUD_PROJECT to point to prod.

Reply:
I see - to my key or just to `prod` verbatim?

Reply:
No set it to our prod project, which I don't think I should type here